### PR TITLE
Feature/global responsive header

### PR DESCRIPTION
**Pull request by diegomateus**

### CODE REVIEW

File: src/assets/css/header.scss [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+}
+
+.desktop {
+  background-color: $header-background;
+  padding: 60px 75px;
+  position: relative;
+  max-width: 170px;
+  top:0;
+  left: 0;
+  -webkit-transition-duration: 500ms;
```
[the remark this answers is not in the document]
Comment (diegomateus) [reply src/assets/css/header.scss:36]: I don't know what I could replace it with

File: src/assets/css/header.scss [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+$hidden-bar: -75px;
+
+.mor-desktop {
+  background-color: $header-background;
+  padding: 60px 75px;
+  position: relative;
+  max-width: 170px;
+  top:0;
+  left: 0;
+  -webkit-transition-duration: 500ms;
```
Comment (pushdev-code) [inline src/assets/css/header.scss:14]:
```suggestion

```

File: src/assets/css/icons.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    content: $instagram-icon;
+}
+  
+.fa-facebook:before {
+    content: $facebook-icon;
+}
+  
+.fa-twitter:before {
+    content: $twitter-icon;
+}
```
Comment (pushdev-code) [inline src/assets/css/icons.scss:25]: You can simplify to:
```suggestion
$close-icon: '\f00d';

$mor-icons: (
  pinterest: '\f0d2',
  instagram: '\f16d',
  facebook: '\f09a',
  twitter: '\f099'
);

.fa {
    font: normal normal normal 18px/1 FontAwesome;
    
    @each $social, $icon in $mor-icons {
      &.fa-#{$social}::before {
        content: $icon;
      }
  }
}
```

File: src/assets/css/header.scss [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+    left: 0;
+    transition-duration: 500ms;
+  }
+
+  @media screen and (max-width: 767px) and (min-width: 320px){
+
+    .button-close {
+      background-color: $line-bgcolor;
+      color: $header-background;
+      cursor: pointer;
```
[the remark this answers is not in the document]
Comment (diegomateus) [reply src/assets/css/header.scss:56]: is not a but, but i put that name like a guide

**Pull request closed without merging**